Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y composer php-fpm php-cli php-gd php-xml php-apcu php-mbstring php-intl apache2 apache2-utils libfcgi0ldbl task-spooler
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN mkdir /standardebooks.org
+RUN mkdir /standardebooks.org/web
+RUN mkdir /standardebooks.org/web/config
+RUN mkdir /standardebooks.org/web/config/ssl
```

Comment:
This can all be combined into one line with mkdir -p

Reply:
https://github.com/standardebooks/web/pull/108/commits/2b4af01b060ead9ba0624717e1460ad145cf5a3b